lib/WP/Functions.php

```diff
@@ -84,26 +95,379 @@ function wpEncodeEmoji() {
     return call_user_func_array('wp_encode_emoji', func_get_args());
   }
 
+  function __() {
+    return call_user_func_array('__', func_get_args());
+  }
+
+  function stripslashesDeep() {
+    return call_user_func_array('stripslashes_deep', func_get_args());
+  }
```

Comment:
So I see this class for the first time and it seems we're using `call_user_func_array` everywhere. I understand it's easier to do for the initial refactoring stage but I also think we really should annotate the methods properly & pass parameters by variables. It will be much more convenient for use in IDEs, for devs when looking into the code, and for PHPStan. Should we create a ticket for that?

Reply:
I agree. This can be the next step.